core/core.go

```diff
@@ -70,15 +70,15 @@ func (h *Handler) info(ctx context.Context) (map[string]interface{}, error) {
 
 func (h *Handler) leaderInfo(ctx context.Context) (map[string]interface{}, error) {
 	var (
-		generatorHeight  *uint64
-		generatorFetched *time.Time
+		generatorHeight  uint64
```

Comment:
If `0` truly means "we don't know", I'd update these eventually:

- https://github.com/chain/chain/blob/stop-pointing/dashboard/src/features/core/reducers.js#L39
- https://github.com/chain/chain/blob/stop-pointing/dashboard/src/features/core/reducers.js#L68
- https://github.com/chain/chain/blob/stop-pointing/dashboard/src/features/core/reducers.js#L131

However, I don't believe Dashboard will break if it gets zero back instead of null. The worst consequence is that replication lag will be "green" instead of "red" for `0`.